WorkflowFunctionAdminBundle/Tests/Security/Authorization/Voter/WorkflowRightVoterTest.php

```diff
@@ -138,11 +141,20 @@ public function provideObject()
     {
         $object0 = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentInterface');
         Phake::when($object0)->getContentType()->thenReturn($this->contentType);
+        Phake::when($object0)->getCreatedBy()->thenReturn($this->username);
         $object1 = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentInterface');
         Phake::when($object1)->getContentType()->thenReturn($this->contentType);
+        Phake::when($object1)->getCreatedBy()->thenReturn($this->username);
 
         $object2 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object2)->getCreatedBy()->thenReturn($this->username);
         $object3 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object3)->getCreatedBy()->thenReturn($this->username);
+        $object4 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object4)->getCreatedBy()->thenReturn('fakeOtherUsername');
+
+        $object5 = Phake::mock('OpenOrchestra\ModelInterface\Model\StatusableInterface');
+        Phake::when($object5)->getCreatedBy()->thenReturn($this->username);
```

Comment:
as the `$object5` is a mock of `StatusableInterface` the method `getCreatedBy` does not exists